deployments/liqo/templates/liqo-clusterid-configmap.yaml

```diff
@@ -9,10 +9,12 @@ metadata:
     {{- include "liqo.labels" $clusterIdConfig | nindent 4 }}
   name: {{ include "liqo.prefixedName" $clusterIdConfig }}
 data:
-  {{- if (not $oldObject) }}
-  CLUSTER_ID: {{ uuidv4 }}
-  {{- else }}
+  {{- if $oldObject }}
```

Comment:
I would suggest to emit a warning in case a clusterIDOverride is set, but a different value is being retrieved from the old object. Silently ignoring it might fool the users.